modules/exploits/multi/ssh/sshexec.rb

```diff
@@ -92,26 +135,28 @@ def initialize
   def execute_command(cmd, opts = {})
     vprint_status("Executing #{cmd}")
     begin
-      Timeout.timeout(3) do
+      Timeout.timeout(5) do
```

Comment:
Magic numbers ftw!

Reply:
Looking back at this..... do we even need it?

Reply:
If we do, not hard coded. Too many Integers around already.